src/demo-app/demo-app/demo-app.scss

```diff
@@ -1,3 +1,19 @@
+// The default indigo-pink theme will load through the index file. This makes the SCSS build faster
+// since Sass doesn't need to build a "prebuilt" theme again.
+@import '../../lib/core/theming/all-theme';
```

Comment:
When I ran it locally it worked as expected? Sass imports the file at build-time and doesn't compile with the `dist/` folder as CWD.